src/app/landing/landing.component.ts

```diff
@@ -1,9 +1,10 @@
-import { Component } from '@angular/core';
+import { Component, ViewEncapsulation } from '@angular/core';
 
 @Component({
   selector: 'app-landing',
   templateUrl: './landing.component.html',
-  styleUrls: ['./landing.component.css']
+  styleUrls: ['./landing.component.css'],
+  encapsulation: ViewEncapsulation.None
```

Comment:
Concider to use ViewEncapsulation.ShadowDom for Landing. It has some limits. But useful for such cases.

You do not need leak of your landing styles to other app components

Reply:
Changed to ShadowDom, I also had to include reset and font-face styles because angular global styles are not duplicated inside shadow dom.